mava/configs/default_ff_ippo.yaml

```
@@ -3,5 +3,5 @@ defaults:
   - arch: anakin
   - system: ppo/ff_ippo
   - network: mlp
-  - env: rware
+  - env: rware_gym
```

Surely this should stay rware?
```suggestion
  - env: rware
```